### PR TITLE
make -checkout: none an optional phase

### PR DESCRIPTION
**Pull request by scbedd**
in `sparse-checkout.yml`

### CODE REVIEW

File: eng/common/pipelines/templates/steps/sparse-checkout.yml [hunk 1]
```diff
@@ -8,9 +8,13 @@ parameters:
       - Name: $(Build.Repository.Name)
         Commitish: $(Build.SourceVersion)
         WorkingDirectory: $(System.DefaultWorkingDirectory)
+  - name: MultipleCheckout
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/sparse-checkout.yml:11]: I'd suggest calling this something like SkipDefaultCheckout.

Comment (scbedd) [reply eng/common/pipelines/templates/steps/sparse-checkout.yml:11]: K.

Comment (scbedd) [reply eng/common/pipelines/templates/steps/sparse-checkout.yml]: @weshaggard  done.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**